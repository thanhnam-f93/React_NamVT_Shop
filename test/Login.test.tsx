import React from "react";
import SignIn from "../src/pages/signin/page";
import renderer from "react-test-renderer";

it("Test Snapshot", () => {
  const component = renderer.create(<SignIn />);
  let tree = component.toJSON();
  expect(component).toMatchSnapshot();
});
it("changes the class when hovered", () => {
  const component = renderer.create(<SignIn />);
  let tree = component.toJSON();
  expect(component).toMatchSnapshot();
});
