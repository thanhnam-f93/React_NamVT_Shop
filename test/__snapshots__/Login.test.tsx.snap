// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `null`;
